heron/executor/src/python/BUILD

```diff
@@ -2,8 +2,15 @@ package(default_visibility = ["//visibility:public"])
 
 load("/tools/rules/pex_rules", "pex_library", "pex_binary")
 
+pex_library(
+    name = "heron-executor-lib",
+    srcs = ["heron_executor.py"],
+    deps = ["//heron/statemgrs/src/python:zk-statemgr-py"],
+    reqs = ["pyyaml==3.10"],
+)
+
 pex_binary(
     name = "heron-executor",
-    srcs = ["heron-executor.py"],
-    reqs = ["pyyaml==3.10"],
+    srcs = ["heron_executor.py"],
+    deps = [":heron-executor-lib"],
```

Comment:
alternatively - you could separate the main into main.py and keep the remaining code as heron-executor.py. In this case, you could compile heron-executor as lib and main.py as pex-binary with heron-executor as lib for linking. Feel free to punt this suggestion if it is not worthwhile.